nav2_planner/include/nav2_planner/planner_server.hpp

```diff
@@ -38,31 +38,75 @@
 
 namespace nav2_planner
 {
-
+/**
+ * @class nav2_planner::PlannerServer
+ * @brief An action server implements behavior tree's ComputePathToPose
```

Comment:
I'd add the same type of blurb: hosts varying plugins... algorithms... to compute plans...

Reply:
Is it okay?
```
/**
 * @class nav2_planner::PlannerServer
 * @brief An action server implements the behavior tree's ComputePathToPose
 * interface and hosts various plugins of different algorithms to compute plans.
 */
```

Reply:
Sure!